.ci/run_fetch_branches_tool.sh

```diff
@@ -21,13 +21,18 @@ set -e
 
 cidir=$(dirname "$0")
 
-github_repository="src/github.com/clearcontainers/tests/cmd/fetchbranches"
-tool_dir="${GOPATH}/${github_repository}"
+tests_repo="github.com/clearcontainers/tests"
+fetchtool_path="$GOPATH/src/$tests_repo/cmd/fetchbranches"
 
-pushd ${tool_dir} 
+if [ ! -d "$fetchtool_path" ];then
+	go get "$tests_repo"
```

Comment:
as you are not trying to build/install the actual repo here, but do proceed to specifically build a sub-part of the repo below, maybe a `go get -d` would be more appropriate?